app/controllers/api/v1/players_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::PlayersController < ApplicationController
+  def index
+    render json: Player.all
```

Comment:
should we sort these by name? `Player.all.order(:name)`

Reply:
I wanted to sort it by last_played but we don't have that info right now.

Reply:
cool no need to sort for now then